jest-playwright.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  // https://github.com/playwright-community/jest-playwright#start-a-server
+  serverOptions: [
+    {
+      command: 'npm run bundle && npx serve -C -l 8000 dist/sdk',
```

Comment:
The watch command wont work right?

This needs to also watch and rebuild for the test watch to work for e2e?

I think if its tough to get that hooked up we can remove watch from e2e... wdyt?

Reply:
Hmmm, I work for changes on the test file, but the bundling with not self-update here, you are right.
I'll remove it then, and we can hook it up if we need it in the future.